parsl/dataflow/dflow.py

```diff
@@ -33,6 +33,8 @@
 from parsl.monitoring.db_logger import get_db_logger
 from parsl.monitoring import app_monitor
 from parsl.monitoring import logging_server
+from parsl.monitoring.web_app import index
```

Comment:
yadu posted a note in slack about handling optional imports - see https://github.com/Parsl/parsl/blob/master/parsl/monitoring/db_logger.py#L7